plugin-java/che-plugin-java-ext-debugger-java-client/src/main/resources/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeLocalizationConstant.properties

```diff
@@ -46,7 +46,7 @@ evaluateExpression = Evaluate Expression
 ############### Messages ################
 debugger.connecting = Connecting to the target VM...
 debugger.disconnecting = Disconnecting from the target VM...
-debugger.connected = Connected to the target VM, address: {0}
+debugger.connected = Connected to the target VM, address: {0}.
```

Comment:
I think you should remove dot at the end of line. See https://docs.google.com/spreadsheets/d/1mNmRVN0jTWBjemb9U7JOvLLCXlvu2Jfa9EwPY2cnguc/edit#gid=0